<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>371</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PIC 10B Intermediate Programming</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PIC 10C Advanced Programming</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Schema A</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Schema B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Overall Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>56</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSpinBox" name="hwBox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hwBox_2">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hwBox_3">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hwBox_4">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hwBox_5">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hwBox_6">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hwBox_7">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hwBox_8">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>211</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSlider" name="hwSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hwSlider_2">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hwSlider_3">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hwSlider_8">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hwSlider_4">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hwSlider_7">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hwSlider_5">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hwSlider_6">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hw1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hw2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hw3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Hw4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Hw5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Hw6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Hw7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Hw8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>191</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Midterm 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="mt1Slider_9">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mt1Box_9">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>191</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Midterm 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="mt2Slider_10">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mt2Box_10">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>191</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Final</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="finalSlider_11">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="finalBox_11">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>hwSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwBox_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwBox_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwBox_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hwBox_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hwSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mt1Slider_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mt1Box_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mt1Box_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mt1Slider_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mt2Slider_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mt2Box_10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mt2Box_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mt2Slider_10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finalSlider_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>finalBox_11</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finalBox_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>finalSlider_11</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
